### ISSUE TITLE
Lock v11.5.0 breaks login where v11.4.0 worked

### ISSUE THREAD

**Issue by MatthewCallis** [auth0/lock#1312]
I have a working integration that currently uses `auth0-lock` v11.4.0 and `auth0-js` v9.3.4, After upgrading to `auth0-lock` v11.5.0 and `auth0-js` v9.4.0 attempting to log in now emits an `authorization_error` of `{ error: "invalid_token", errorDescription: "Invalid access_token" }`. There does not appear to be anything that should be breaking in the minor version updates, but nothing else has changed, and reverting to the older version continues to work.

Is there a breaking change in this version?

**Comment by luisrudge** [comment]
How are you trying to log in with your application? Can you show me your configuration so I can test?
In v11.5.0, we're now validating access tokens according to spec, which we weren't before. So, you should only get this message when your access_token is not valid.

https://github.com/auth0/auth0.js/pull/709/files#diff-2f11667c8f671044416e38e4ceb9d7c8R259
https://github.com/auth0/idtoken-verifier/blob/master/src/index.js#L274

**Comment by MatthewCallis** [comment]
I'm logging in with Google OAuth in both cases. Below is the Lock config:

```
const options = {
  auth: {
    redirectUrl: process.env.AUTH0_REDIRECT_URI,
    responseType: 'token',
  },
  closable: false,
  rememberLastLogin: false,
  languageDictionary: {
    title: 'Title',
  },
  socialButtonStyle: 'small',
  theme: {
    logo: 'data:image/png;base64,...',
    primaryColor: '#cc3833',
  },
  additionalSignUpFields: [
    {
      name: 'name',
      icon: 'https://secure.gravatar.com/avatar/af3c661a01db6265b323e253699e8cd4?s=80&d=mm&r=x',
      placeholder: 'your full name',
    },
  ],
  initialScreen: 'login',
};

const Lock = new Auth0Lock(
  process.env.AUTH0_CLIENT_ID,
  process.env.AUTH0_DOMAIN,
  options
);

Lock.on('authorization_error', (error) => {
  console.error('Auth0 Authorization Error', error);
  window.location.assign('/logout');
});

Lock.on('unrecoverable_error', (error) => {
  console.error('Auth0 Unrecoverable Error', error);
  window.location.assign('/logout');
});
```

**Comment by luisrudge** [comment]
Thanks! I found the issue. Your client is not set as OIDC, right? So your id_token doesn't have the `at_hash` claim that we [need](https://github.com/auth0/auth0.js/blob/master/src/web-auth/index.js#L257) to validate the access_token. I'll submit a patch shortly!

**Comment by MatthewCallis** [comment]
I'm not sure about OIDC, where can I confirm that for you in the dashboard?

**Comment by luisrudge** [comment]
The fix is here: https://github.com/auth0/auth0.js/pull/718 I plan to release this shortly, just waiting for review etc

**Comment by MatthewCallis** [comment]
Thank you @luisrudge

**Comment by luisrudge** [comment]
Hey @MatthewCallis we just published a version with the fix! Please try 11.5.1
